pyproject.toml

```diff
@@ -1,47 +1,203 @@
 [build-system]
-requires = ["setuptools>=40.8.0", "wheel", "cython"]
-build-backend = "setuptools.build_meta:__legacy__"
+requires = ["setuptools>=64", "setuptools_scm>=8", "wheel"]
+build-backend = "setuptools.build_meta"
 
+[project]
+authors = [{name = "The HyperSpy Community"}]
+classifiers = [
+    "Development Status :: 4 - Beta",
+    "Intended Audience :: Developers",
+    "Intended Audience :: Science/Research",
+    "License :: OSI Approved :: GNU General Public License v3 (GPLv3)",
+    "Operating System :: OS Independent",
+    "Programming Language :: Python :: 3.8",
+    "Programming Language :: Python :: 3.9",
+    "Programming Language :: Python :: 3.10",
+    "Programming Language :: Python :: 3.11",
+    "Programming Language :: Python :: 3.12",
+    "Topic :: Scientific/Engineering",
+    "Topic :: Scientific/Engineering",
+    "Topic :: Scientific/Engineering :: Physics",
+    "Topic :: Software Development :: Libraries",
+]
+dependencies = [
+    "cloudpickle",
+    "dask[array]>=2021.3.1",
+    "importlib-metadata>=3.6",
+    "jinja2",
+    "matplotlib>=3.1.3",
+    "natsort",
+    "numpy>=1.20.0",
+    "packaging",
+    "pint>=0.10",
+    "pooch",
+    "prettytable>=2.3",
+    "python-dateutil>=2.5.0",
+    "pyyaml",
+    "requests",
+    "rosettasciio[hdf5]",
+    "scikit-image>=0.18",
+    "scipy>=1.5.0",
+    "sympy>=1.6",
+    "tqdm>=4.9.0",
+    "traits>=4.5.0",
+]
+description = "Multidimensional data analysis toolbox"
+dynamic = ["version"]
+keywords = [
+    "EDX",
```

Comment:
The keywords contain a selective set of the methods that can be addressed with hyperspy and its extensions. I would not throw them out, but add at least a selection of other methods:
- 4DSTEM
- EBSD / electron backscatter diffraction
- CL / cathodoluminescence
- PL / photoluminescence
- off-axis electron holography